src/components/Debug.vue

```diff
@@ -37,7 +40,23 @@ export default {
     clearContent: function () {
       this.terminalOutput = ''
       this.terminalInput = ''
+      this.executionResult = ''
+    },
+    executeCode: function () {
+      if (this.terminalInput !== '') {
+        const success = Math.random() < 0.5
```

Comment:
Can we emulate success or not in the store instead here?

this.$store.commit('runDebug', this.terminalInput )